wallet/src/main/scala/org/bitcoins/wallet/internal/TransactionProcessing.scala

```diff
@@ -275,21 +275,9 @@ private[wallet] trait TransactionProcessing extends WalletLogger {
           insertTransaction(transaction, blockHashOpt)
         else Future.unit
 
-      // unreserved outputs now they are in a block
-      outputsToUse = blockHashOpt match {
-        case Some(_) =>
-          outputsBeingSpent.map { out =>
-            if (out.state == TxoState.Reserved)
-              out.copyWithState(TxoState.PendingConfirmationsReceived)
-            else out
-          }
-        case None =>
-          outputsBeingSpent
-      }
```

Comment:
what is the reason for doing this? If we remove this then the utxo won't change to un-reserved

Reply:
This diff is gone now and moved to #2919